operations/boot_orchestration/Boot_UANs.md

```diff
@@ -61,69 +60,74 @@ UAN boot images and a BOS session template have been created. See [Create UAN Bo
 
     If this occurs, repeat the BOS command.
 
-2.  Verify that the Day Zero patch was applied correctly during [Create UAN Boot Images](../image_management/Create_UAN_Boot_Images.md). Skip this step if the patch has already been verified.
-
-    1.  SSH into a newly booted UAN.
-
-        ```bash
-        ncn-m001# ssh uan01-nmn
-        ```
-
-    2.  Verify that the DVS RPM versions match what exists in the 1.4.0-p2/rpms directory.
-
-        ```bash
-        uan01# rpm -qa | grep 'cray-dvs.*2.12' | sort
-        ```
-
-        Example output:
-
-        ```
-        cray-dvs-compute-2.12_4.0.102-7.0.1.0_8.1__g30d29e7a.x86_64
-        cray-dvs-devel-2.12_4.0.102-7.0.1.0_8.1__g30d29e7a.x86_64
-        cray-dvs-kmp-cray_shasta_c-2.12_4.0.102_k4.12.14_197.78_9.1.58-7.0.1.0_8.1__g30d29e7a.x86_64
-        ```
-
-    3.  Log out of the UAN.
-
-        ```bash
-        uan01# exit
-        ```
-
-3.  Retrieve the BOS session ID from the output of the previous command.
+1. Retrieve the BOS session ID from the output of the `cray bos session create` command in the previous step.
 
     ```bash
-    ncn-m001# export BOS_SESSION=$(jq -r '.links[] | select(.rel=="session") | .href' session.json | cut -d '/' -f4)
+    ncn# BOS_SESSION=$(jq -r '.links[] | select(.rel=="session") | .href' session.json | cut -d '/' -f4) ; echo $BOS_SESSION
+    ```
+
+    Example output:
 
-    ncn-m001# echo $BOS_SESSION
+    ```text
     89680d0a-3a6b-4569-a1a1-e275b71fce7d
     ```
 
-4.  Retrieve the Boot Orchestration Agent \(BOA\) Kubernetes job name for the BOS session.
+1. Retrieve the Boot Orchestration Agent \(BOA\) Kubernetes job name for the BOS session.
 
     ```bash
-    ncn-m001# BOA_JOB_NAME=$(cray bos session describe $BOS_SESSION --format json | jq -r .boa_job_name)
+    ncn# BOA_JOB_NAME=$(cray bos session describe $BOS_SESSION --format json | jq -r .boa_job_name)
     ```
 
-5.  Retrieve the Kubernetes pod name for the BOA assigned to run this session.
+1. Retrieve the Kubernetes pod name for the BOA assigned to run this session.
 
     ```bash
-    ncn-m001# BOA_POD=$(kubectl get pods -n services -l job-name=$BOA_JOB_NAME \
-    --no-headers -o custom-columns=":metadata.name")
+    ncn# BOA_POD=$(kubectl get pods -n services -l job-name=$BOA_JOB_NAME --no-headers -o custom-columns=":metadata.name")
```

Comment:
ncn# doesn't seem right. I don't think this will be something you run from a storage node

Reply:
Addressing this with https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-4815